src/clients/FtpClient.h

```diff
@@ -109,7 +109,7 @@ class DataChannel: public Ftp::Channel
     bool read_pending;
 
     /// amount of message payload/body received so far
-    int64_t payloadSeen = 0;
+    size_t payloadSeen = 0;
```

Comment:
Normally, received size is not limited to what we can store inside one continues memory block. If you are sure this has to be size_t, please add a C++ comment to explain this exceptional case. Otherwise, please use uint64_t.

Reply:
Ok.